python/bindir/cloud-grab-dependent-library-versions

```diff
@@ -27,7 +27,7 @@ def runCmd(cmds):
     stdout, stderr = process.communicate()
     if process.returncode != 0:
         raise Exception(stderr)
-    return stdout
+    return stdout.decode('utf-8')
```

Comment:
On depLibraries I think that it would be necessary also to add python3.

depLibraries = ['python', 'pythom3', ...]

Reply:
@GabrielBrascher centos8 has python2 as python2 python3 as python36 and centos7 has python2 as python and python3 as python3 so I've added condition for that in the script.
Though I'm not sure where that script is used. Could see it getting installed
https://github.com/apache/cloudstack/pull/4068/commits/41a69cb3d985b55f76d1f47b51aae34b60e61763